### ISSUE TITLE
Auto complete Dropdown not closing in ie 11

### ISSUE THREAD

**Issue by Sriram143** [angular/material#3245]

**Comment by epelc** [comment]
@Sriram143 What version are you using? Have you tried the latest master?

**Comment by Sriram143** [comment]
Yes i am using latest master. still not closing when i set minlenght=0

**Comment by Abrissirba** [comment]
I have the same problem on both select and the new menu component in both IE and firefox. I think the problem is related to 6911506712a9ef71ebefbcd63f427e4325d82001 where the scroll lock was updated. 

When I tried the components in the demo page, $window.getComputedStyle(element[0])  returned "" in chrome which works fine. However in both firefox and IE it returns" auto" which makes the substraction with 1 to return NaN. Then the z-index will be set to 99 which will placethe backdrop that listens for clicks and that it will close the dropdown, select or menu  behind the scroll lock element. Because of this the click event on the backdrop will never be triggered and the dropdown will not be closed. I'm not sure why the z-index is hard-coded to 99 if the z-index is NaN. How would one handle when the z-index is set to auto. Is it possible to get the actual z-index in another way?

**Comment by webmandman** [comment]
Same problem. And after updating from 0.9.8 to 0.10.0 it did improve a bit.

This screencapture is at 0.9.8
![autocompletedropdownissues](https://cloud.githubusercontent.com/assets/991677/8168902/fadc4746-135a-11e5-8512-1118067ad62c.gif)

And with 0.10.0
![autocompletedropdownissues 0 10 0](https://cloud.githubusercontent.com/assets/991677/8168956/4a99f472-135b-11e5-8ee1-09ed07b68730.gif)

Still problematic. I typed "santa" and it stopped at "san", then I typed "ta" and it stopped at "t". Nasty delays going on here. Then ofcourse the suggestions would disappear.
